packages/tools/client-debugger/client-debugger-view/package.json

```diff
@@ -52,32 +52,32 @@
     "@fluentui/react-hooks": "^8.6.12",
     "@fluentui/react-icons-mdl2": "^1.3.22",
     "@fluentui/react-tabs": "^9.0.6",
-    "@fluid-tools/client-debugger": ">=2.0.0-internal.1.2.0 <2.0.0-internal.3.0.0",
-    "@fluidframework/container-definitions": ">=2.0.0-internal.1.2.0 <2.0.0-internal.3.0.0",
-    "@fluidframework/container-loader": ">=2.0.0-internal.2.1.0 <2.0.0-internal.3.0.0",
-    "@fluidframework/container-runtime": ">=2.0.0-internal.2.1.0 <2.0.0-internal.3.0.0",
-    "@fluidframework/core-interfaces": ">=2.0.0-internal.1.2.0 <2.0.0-internal.3.0.0",
-    "@fluidframework/counter": ">=2.0.0-internal.2.1.0 <2.0.0-internal.3.0.0",
-    "@fluidframework/datastore": ">=2.0.0-internal.2.1.0 <2.0.0-internal.3.0.0",
-    "@fluidframework/driver-definitions": ">=2.0.0-internal.1.2.0 <2.0.0-internal.3.0.0",
-    "@fluidframework/map": ">=2.0.0-internal.2.1.0 <2.0.0-internal.3.0.0",
+    "@fluid-tools/client-debugger": ">=2.0.0-internal.2.3.0 <2.0.0-internal.3.0.0",
+    "@fluidframework/container-definitions": ">=2.0.0-internal.2.3.0 <2.0.0-internal.3.0.0",
+    "@fluidframework/container-loader": ">=2.0.0-internal.2.3.0 <2.0.0-internal.3.0.0",
+    "@fluidframework/container-runtime": ">=2.0.0-internal.2.3.0 <2.0.0-internal.3.0.0",
+    "@fluidframework/core-interfaces": ">=2.0.0-internal.2.3.0 <2.0.0-internal.3.0.0",
+    "@fluidframework/counter": ">=2.0.0-internal.2.3.0 <2.0.0-internal.3.0.0",
+    "@fluidframework/datastore": ">=2.0.0-internal.2.3.0 <2.0.0-internal.3.0.0",
+    "@fluidframework/driver-definitions": ">=2.0.0-internal.2.3.0 <2.0.0-internal.3.0.0",
+    "@fluidframework/map": ">=2.0.0-internal.2.3.0 <2.0.0-internal.3.0.0",
     "@fluidframework/protocol-definitions": "^1.1.0",
-    "@fluidframework/sequence": ">=2.0.0-internal.2.1.0 <2.0.0-internal.3.0.0",
-    "@fluidframework/shared-object-base": ">=2.0.0-internal.1.2.0 <2.0.0-internal.3.0.0",
+    "@fluidframework/sequence": ">=2.0.0-internal.2.3.0 <2.0.0-internal.3.0.0",
+    "@fluidframework/shared-object-base": ">=2.0.0-internal.2.3.0 <2.0.0-internal.3.0.0",
     "re-resizable": "6.9.9",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
     "react-json-view": "^1.21.3"
   },
   "devDependencies": {
-    "@fluid-example/collaborative-textarea": ">=2.0.0-internal.2.1.0 <2.0.0-internal.3.0.0",
-    "@fluid-tools/webpack-fluid-loader": ">=2.0.0-internal.1.2.0 <2.0.0-internal.3.0.0",
+    "@fluid-example/collaborative-textarea": ">=2.0.0-internal.2.3.0 <2.0.0-internal.3.0.0",
+    "@fluid-tools/webpack-fluid-loader": ">=2.0.0-internal.2.3.0 <2.0.0-internal.3.0.0",
     "@fluidframework/build-common": "^1.1.0",
     "@fluidframework/build-tools": "^0.7.0",
     "@fluidframework/eslint-config-fluid": "^2.0.0",
-    "@fluidframework/fluid-static": ">=2.0.0-internal.1.2.0 <2.0.0-internal.3.0.0",
+    "@fluidframework/fluid-static": ">=2.0.0-internal.2.3.0 <2.0.0-internal.3.0.0",
     "@fluidframework/test-utils": "^1.3.0",
```

Comment:
Shouldn't this be updated to `"@fluidframework/test-utils": ">=2.0.0-internal.2.3.0 <2.0.0-internal.3.0.0",`?